hack/test.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 echo "Running go tests..."
-go test -cover -v -timeout 60s `go list ./...  | grep -v vendor`
+go test -cover -v -timeout 60s `go list ./...  | grep -v vendor | grep -v integration_tests`
```

Comment:
You can also require a special tag on the integration tests, so they don't run without it.